src/views/new-evaluation/newEvaluation.js

```diff
@@ -0,0 +1,302 @@
+        <li >
+            <input type="radio" name="level" value = "S1">
+        </li>
+        <li >
+            <input type="radio" name="level" value = "S2">
+        </li>
+        <li >
+            <input type="radio" name="level" value = "S3">
+        </li>
+        <div class="clearfix"></div>
+    </ul>`
+}
+
+const constructorTechnicalLevelPicker = function(option = {}){
+    return `${technicalLevelPickerHeader(option.HeaderTitle)}
+    ${technicalLevelPickerTop(option.Titles)}
+    ${technicalLevelPickerBottom({})}
+    `
+}
+
+const technicalLevelPicker = function (option = {}){
+    return `<div class="radio-boxes">
+   ${constructorTechnicalLevelPicker(option)}
+    <br>			
+    </div>
+`
+} 
+
+const textAreaBox = function(option = {}){
+    return `<div class="user-textarea">
+        <h3>${option.HeaderTitle}</h3>
+        <textarea placeholder="${option.Placeholder}"required></textarea>
+    </div>`
+}
+
+
+const textArea = function (option = {}){
+    const result = []
+    for(var i=0;i<option.HeaderTitle.length;i++){
+        result.push(textAreaBox({HeaderTitle: option.HeaderTitle[i], Placeholder: option.Placeholder[i]}))
+    }
+    return result.join('')
+}
+
+
+const selectConstructor = function(option= {}){
+    const result = []
+    result.push(`<option selected disabled hidden>Evaluation</option>`)
+    for (var i=0;i<option.Values.length;i++){
+        result.push(`<option value = "0"> ${option.Values[i]}</option>`)
+    }
+    return result.join('')
+}
+
+const dropDownConstructor = function (option={}){
+    const result = []
+    for (var i=0;i<option.selectName.length;i++){
+        result.push(`
+        <li class="legend-box-drop">
+            <label for="${option.selectName[i]}"> ${option.labelTitle[i]}</label>
+            <select name="${option.selectName[i]}">
+            ${selectConstructor(option.sContent)}
+            </select>
+        </li>
+        `)
+    }
+    return result.join('')
+}
+
+const newEvaluationForm = function (option = {}){
+    var result = []
+    for(var i=0;i<option.Data.length;i++){
+        result.push(`<form class="legend-box" id="${option.boxID[i]}">
+        <fieldset>
+            <legend>${option.Title[i]}</legend>
+            <ul class="legend-box-list">
+                ${dropDownConstructor(option.Data[i])}
+            </ul>
+        </fieldset>
+    </form>`)
+    }
+    return result.join('')
+}
+
+const dropDownSelector = function (option = {}){
+
+}
+
+const submitButton =function (option = {}){
+    return `
+    <div class= "submit-button">
+        <form action="Evaluation.html">
+            <input  type="submit" name="button" value="Submit">
+        </form>
+    </div>`
+}
+
+const Footer = function(option = {}){
+    return `	
+    <footer class="footer footerT" >
+        <p>${option.Footer}</p>
+    </footer>`
+}
+
+const PageGrid = function (option = {}){
+    return `	<div class = "mainBlock">
+    ${candidateDetailsForm(option.cContent)}
```

Comment:
Same comment about being more explicit about your naming. It's a bit confusing to tell what `cContent` / `tlContent` / `taContent` etc mean outside of context. Do keep in mind that these options might eventually come from an API endpoint that maybe grabs the info from a database, so you want these to make sense for everyone, not just yourself. ;)